src/tokens/evo-core.css

```diff
@@ -109,4 +109,15 @@
     --font-size-giant-4: var(--font-size-64);
     --font-weight-regular: 500;
     --font-weight-bold: 700;
+    --spacing-100: 8px;
+    --spacing-25: calc(var(--spacing-100) * 0.25);
```

Comment:
May be computing the value `calc(var(--spacing-100) * 0.25)` to `2px` might be better here. any thoughts?

Reply:
Yes, `2px` here would be fine. Don't think we need to be too clever.